pybb/compat.py

```diff
@@ -2,7 +2,44 @@
 from __future__ import unicode_literals
 import django
 from django.conf import settings
+from pybb import defaults
 
+if defaults.PYBB_USE_DJANGO_MAILER:
+    try:
+        from mailer import send_html_mail, send_mail
+    except:
+        pass
+
+try:
+    send_mass_html_mail
+except:
```

Comment:
Please use the [narrower `except ImportError`](http://stackoverflow.com/a/14797508/180709).